content/en/about/ecosystem/index.md

```diff
@@ -1,14 +1,16 @@
 ---
 title: Ecosystem
-description: Ecosystem.
+description: A list of Istio providers, professional services companies and integrations.
```

Comment:
nit:
```suggestion
description: A list of Istio providers, professional service companies and integrations.
```

Reply:
Plural is actually correct wording in this case (see Google autocomplete!); the category is "pro services" on the page too.